src/lambda-powertools-layer.ts

```diff
@@ -122,6 +127,84 @@ export class LambdaPowertoolsLayer extends lambda.LayerVersion {
   }
 }
 
+export class LambdaPowertoolsLayerPythonV3 extends lambda.LayerVersion {
```

Comment:
I'd suggest to move it to a dedicated file `lambda-powertools-layer-pythonv3.ts` to have a clean separation, we might back port it at some point.

Reply:
Good point! Done